Gemfile

```diff
@@ -50,6 +50,8 @@ gem "bootsnap", require: false
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 # gem 'image_processing', '~> 1.2'
 
+gem "csv"
```

Comment:
Potentially silly question but how come you've added CSV to the Gemfile? 🤔

Is that a best practice thing, or some other reason to do with Ruby 3.3.1?

Reply:
Oh cool, thanks @gms-gs 👍🏻